#! /bin/sh /usr/share/dpatch/dpatch-run
## 60_version.dpatch by Romain Francoise <rfrancoise@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix version.

@DPATCH@
diff -urNad --exclude=CVS --exclude=.svn ./VERSION /tmp/dpep-work.XGAkv1/libpcap0.8-0.9.3/VERSION
--- ./VERSION	2005-07-11 22:09:47.000000000 +0200
+++ /tmp/dpep-work.XGAkv1/libpcap0.8-0.9.3/VERSION	2005-07-16 14:22:35.000000000 +0200
@@ -1 +1 @@
-0.9.2
+0.9.3
